app/javascript/packs/util/util.js

```diff
@@ -0,0 +1,10 @@
+//takes in the ID of a specific category that we have selected and returns the questions associated with that category
```

Comment:
Can we rename this to `utils/QuestionsFilter`? To stay consistent with: https://github.com/calblueprint/RMI/pull/178/files#diff-73af159e943dba6bae271720b5301b79